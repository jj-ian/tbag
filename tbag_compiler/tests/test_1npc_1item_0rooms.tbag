npc {
	string name;
	string color;
}

npc Cat {
	name = "Tubbs";
	color = "white";
}

item {
	string name;
}

item Purse {
	name = "fluffy purse";
}

true {
	print("You have a ");
	print(Purse.name);
	print(". There's a cat here. Its name is ");
	print(Cat.name);
	print(" and it's ");
	print(Cat.color);

	endgame;
}
