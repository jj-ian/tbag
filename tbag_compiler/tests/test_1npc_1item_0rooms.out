You have a fluffy purse. There's a cat here. Its name is Tubbs and it's white